src/terrama2/impl/DataAccessorGrADS.cpp

```diff
@@ -903,6 +934,29 @@ void terrama2::core::DataAccessorGrADS::writeVRTFile(terrama2::core::GrADSDataDe
                                                      DataSetPtr dataset) const
 {
 
+  try
+  {
+    int numBands = getNumberOfBands(dataset);
+    if(numBands > 0)
+      descriptor.tDef_->numValues_ = numBands;
+  }
+  catch(UndefinedTagException e)
+  {
+    // In case the didn't specify the number of bands it will use the number of bands in the file
+  }
+
+  try
+  {
+    double multiplier = getValueMultiplier(dataset);
+    descriptor.zDef_->numValues_ = 1;
+    descriptor.zDef_->values_.clear();
+    descriptor.zDef_->values_.push_back(multiplier);
```

Comment:
don't use zDef for bands, create a new field for that.
as zDef is for vertical information it can cause confusion.